packages/wp-story-editor/src/style.css

```diff
@@ -26,6 +26,15 @@
   display: none;
 }
 
+body.js.edit-story #wpcontent {
+  position: fixed;
+  width: calc(100% - 160px);
+}
+
+body.js.edit-story.folded #adminmenuback {
+  top: 32px;
```

Comment:
should `body.js.edit-story.folded` be `top: 32` or `right: 32` 🤔

Reply:
#adminmenuback is the div behind the #adminmenuwrap (the menu bar that holds the plugins), which makes the #adminmenuwrap look like its the full length of the page.  #adminmenuback covers up the #wpadminbar (menu bar at the top of the screen) with this fix so I had to move #adminmenuback down by `32px` (the height of #wpadminbar) so that it doesn't cover up #wpadminbar. 😅 it is a little confusing.

![2022-05-24_13-49-47](https://user-images.githubusercontent.com/14057928/170100192-5dca4c6f-ac05-4958-b2f8-3cbd43193af0.png)